autoware_launch/config/system/system_error_monitor/system_error_monitor.param.yaml

```diff
@@ -32,10 +32,11 @@
         /autoware/planning/node_alive_monitoring: default
```

Comment:
@ito-san Could you check if parameters sync of system looks good to you.

Reply:
I'll merge the PR now since the change looks good to me s far as I understand.
Let me know if you have any comments. :pray: